build.sh

```diff
@@ -42,7 +42,7 @@ fi
 
 if [ "${DEB8_HACK}" = "1" ]; then
     # Debian 8
-    pip install -t ${PIP_TARGET_DIR} rpy2==2.8.5
+    pip install -t ${PIP_TARGET_DIR} "rpy2==2.8.5"
```

Comment:
Heh, what's this all about? Is the `==` interpreted specially or something?

Reply:
Yes, that seems to be the case with bash (not entirely surprisingly). On reflection, `==` always seemed like a dodgy thing to have unescaped (for all I know it's interpreted as an assignment of a variable called `=`...).

Reply:
[FWIW, I couldn't replicate this on my machine, but it might be that `=` has some semantics in the Windows Unix environment thingy that I'm unaware of. Either way, I think escaping `==` is a sound plan.]

Reply:
Yeah, no worries.